tests/end2end/test_hints_html.py

```diff
@@ -22,12 +22,17 @@
 import os
 import os.path
 import textwrap
+import string
+import functools
+import operator
 
 import attr
 import yaml
 import pytest
 import bs4
 
+import qutebrowser.browser.hints
```

Comment:
FWIW `from qutebrowser.browser import hints` is okay - it's just things like `from qutebrowser.browser.hints import HintManager` I try to avoid.

Reply:
Ah, that makes a lot more sense now, so modules are fine :)

In this specific case, I'd prefer to leave it as-is, as I don't like taking up the name 'hints' in a hint testing file (and I'm only using it once anyway).